engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -152,7 +153,7 @@ public void saveScreenshot() {
             return;
         }
 
-        FBO sceneFinal = buffersManager.getFBO("sceneFinal");
+        FBO sceneFinal = buffersManager.getFBO(DefaultDynamicFBOs.Final.getResourceUrn());
         int width = sceneFinal.width();
         int height = sceneFinal.height();
```

Comment:
I think we can now skip retrieving the FBO from the manager and simply write:

int width = DefaultDynamicFBOs.Final.width();
int height = DefaultDynamicFBOs.Final.height();

No?